website/docs/job-schedulers/aws-managed-airflow.md

```diff
@@ -52,7 +57,10 @@ The following components are provisioned in your environment:
   - An EMR virtual cluster registered with the newly created EKS
   - A S3 bucket with DAG code
 
```

Comment:
Here we can also mention about our install.sh script which can be used alternatively or update this section